### PR TITLE
Use Marcel to detect the correct content_type

### PR DESCRIPTION
**Pull request by randsina**
Related to the issue #122.
`ActiveStorage` uses `Marcel` gem to detect `content_type` of an attachment.
Idea is to use the same gem for validations.

https://github.com/rails/rails/blob/9053b8f77ffdb28f2494d773cec6fa1a84eceabe/activestorage/app/models/active_storage/blob/identifiable.rb#L22

### CODE REVIEW

File: lib/active_storage_validations/matchers/content_type_validator_matcher.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@ def allowed_types
       end
 
       def rejected_types
-        @rejected_types || (Mime::LOOKUP.keys - allowed_types)
+        @rejected_types || (Marcel::TYPES.keys - allowed_types)
```
Comment (randsina) [inline lib/active_storage_validations/matchers/content_type_validator_matcher.rb:60]:
```ruby
Mime::LOOKUP.keys.count
# => 45
```
```ruby
Marcel::TYPES.keys.count
# => 892
```
It influences the speed of tests

Comment (igorkasyanchuk) [reply lib/active_storage_validations/matchers/content_type_validator_matcher.rb:60]: how big is the impact?

Comment (randsina) [reply lib/active_storage_validations/matchers/content_type_validator_matcher.rb:60]: Doesn't look like a big impact for
```sh
$ BUNDLE_GEMFILE=gemfiles/rails_6_1.gemfile bundle exec rake test
```
before:
```
Finished in 5.077188s, 12.8024 runs/s, 23.0443 assertions/s.
65 runs, 117 assertions, 0 failures, 0 errors, 0 skips
```
after
```
Finished in 5.293181s, 13.0356 runs/s, 22.6707 assertions/s.
69 runs, 120 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by igorkasyanchuk** [comment]
merged, thanks 👍 
I'll release a new version soon

File: lib/active_storage_validations/matchers/content_type_validator_matcher.rb (hunk 1 above)
Comment (IvanShamatov) [reply lib/active_storage_validations/matchers/content_type_validator_matcher.rb:60]: https://github.com/igorkasyanchuk/active_storage_validations/issues/127

**Review by IvanShamatov — COMMENTED** [review]
it definitely affected my suite

**Pull request merged**